code/datums/effects/tether.dm

```diff
@@ -41,6 +41,11 @@
 	if (isnull(tethered))
 		return
 
+	if(isStructure(tethered.affected_atom))//we are attached to a structure, shouldnt move it (too heavy)
+		var/obj/structure/Big_boi = tethered.affected_atom
```

Comment:
nitpick but snake_case is not capitalized throughout. Also this variable could be named way better.

Reply:
Oh and make sure to put your ckey next to the :cl: tag